packages/core/src/actions/getUTXOs.ts

```diff
@@ -0,0 +1,34 @@
+import { UTXOsFetchError } from '../errors/utxo.js'
+import type { UTXOSchema } from '../transports/types.js'
+import type { Account } from '../types/account.js'
+import type { Chain } from '../types/chain.js'
+import type { Client } from '../types/client.js'
+import type { UTXO } from '../types/transaction.js'
+import type { Transport } from '../types/transport.js'
+
+export type GetUTXOsParameters = {
+  address: string
+  minValue?: number // get utxos add up to this value
+}
+
+export type GetUTXOsReturnType = Array<UTXO>
+
+export async function getUTXOs<
+  C extends Chain | undefined,
+  A extends Account | undefined = Account | undefined,
+>(
+  client: Client<Transport, C, A, UTXOSchema>,
+  { address, minValue }: GetUTXOsParameters
+): Promise<GetUTXOsReturnType> {
+  try {
+    return client.request({
+      method: 'getUTXOs',
+      params: {
+        address,
+        minValue,
+      },
+    })
+  } catch (_error) {
+    throw new UTXOsFetchError({ address })
```

Comment:
If we catch an error here, we wouldn't return the real reason why we failed, like when there is not enough balance and we really need to do that, because BE will have logic based on the errors we return.

Reply:
Actually, since we are throwing errors from the method handlers, I don't think we need to catch the errors here, just like with `getBalance`